docs/source/users/splash.md

```diff
@@ -0,0 +1,254 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+kernelspec:
+  display_name: pyrealm_python3
+  language: python
+  name: pyrealm_python3
+---
+
+# The `splash` submodule
+
+The {mod}`~pyrealm.splash` module provides the SPLASH v1.0 model for estimating **soil
+moisture, actual evapotranspiration (AET) and soil water runoff** for a site
```

Comment:
I think `surface runoff` instead of `soil water runoff` would be clearer here because I think SPLASH does not consider any below ground (=soil) water flows?